### PR TITLE
Updates to support CentOS packaging and configuration differences

### PR DESCRIPTION
**Pull request by vitroth**
Part of #ENGT-8709

reviewer: @shr3kst3r 

Let me know what you think about squashing these commits vs leaving them as-is...

### CODE REVIEW

File: recipes/service.rb [hunk 1]
```diff
@@ -10,6 +10,10 @@
 
 # define the service
 service "postgresql" do
+  case node[:platform]
+  when 'redhat', 'centos'
+    service_name "postgresql-9.4"
```
Comment (shr3kst3r) [inline recipes/service.rb]: Would it be worth putting `node["postgresql"]["version"]` here?  Something like `service_name "postgresql-#{node["postgresql"]["version"]}".

**Comment by shr3kst3r** [comment]
I would probably go with squashing them.

**Comment by vitroth** [comment]
@shr3kst3r PTAL at the update, then I'll squash

**Comment by shr3kst3r** [comment]
LGTM

**Comment by vitroth** [comment]
Tested clean, merging.

**Pull request merged**